manifests/repo.pp

```diff
@@ -60,99 +63,96 @@
         $repo_url = "${base_url}/${repo_name}/${osname}"
         $distribution =  $::lsbdistcodename
 
-        file { $normalized_name:
-          ensure  => file,
-          path    => "/etc/apt/sources.list.d/${normalized_name}.list",
-          mode    => '0644',
-          content => template('packagecloud/apt.erb'),
-        }
+        apt::source {"${normalized_name}":
+          location => "${repo_url}",
+          repos    => "${component}",
+          key      => {
+            'id'     => '418A7F2FB0E1E6E7EABF6FE8C2E73424D59097AB',
+            'server' => 'pgp.mit.edu'
```

Comment:
Unfortunately, we can't use `pgp.mit.edu`. We used to use this in the past, but it goes down very, very often and it breaks most of our user's CI/CD/deploys when that happens.